src/Components/ImgList.jsx

```diff
@@ -2,14 +2,14 @@ import React from "react";
 import Img from "./Img";
 
 const ImgList = props => {
-  const arraylist = props.array.urls;
+  const arraylist = props.array;
```

Comment:
Since now we're passing in just the array and not the whole document, we don't need to get the `urls` property anymore.